modules/apps/users-admin/users-admin-impl/src/main/java/com/liferay/users/admin/internal/exportimport/data/handler/OrganizationStagedModelDataHandler.java

```diff
@@ -101,6 +102,27 @@ public String getDisplayName(Organization organization) {
 		return organization.getName();
 	}
 
+	@Override
+	public boolean validateReference(
+		PortletDataContext portletDataContext, Element referenceElement) {
+
+		long companyId = GetterUtil.getLong(
+			referenceElement.attributeValue("company-id"));
+
+		String uuid = GetterUtil.getString(
+			referenceElement.attributeValue("uuid"));
+
```

Comment:
please inline them so we don't need to think about: 
* the order of them (because in the fetch method are uuid, companyId so probably also the declaration should reflect it)
* if the empty line between them should be there or not (I think not)

😄